src/signals/incident/components/IncidentWizard/index.js

```diff
@@ -1,21 +1,27 @@
-/**
-*
-* IncidentWizard
-*
-*/
-
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Route } from 'react-router-dom';
 import { Wizard, Steps, Step } from 'react-albus';
+import { Heading, themeSpacing } from '@datapunt/asc-ui';
+import styled from 'styled-components';
 
 import LoadingIndicator from 'shared/components/LoadingIndicator';
 
 import IncidentForm from '../IncidentForm';
 import IncidentPreview from '../IncidentPreview';
 import onNext from './services/on-next';
 
-import './style.scss';
+const StyledH1 = styled(Heading)`
+  margin-top: ${themeSpacing(6)};;
+  margin-bottom: 30px;
+
+  @media screen and (max-width: 540px) {
```

Comment:
There is no need to need to define a media query here; the `Heading` component already defines the font-size for specific breakpoints in the theme configuration.

Reply:
I thought that it wasn't working ub I see now it is

Reply:
gedaan